web_duplicate_visibility/static/src/js/duplicate_visibility.js

```diff
@@ -0,0 +1,44 @@
+odoo.define('web.DuplicateVisibility',function (require) {
+    "use strict";
+
+    var core = require('web.core');
+    var Sidebar = require('web.Sidebar');
+    var FormView = require('web.FormView');
+
+    var _t = core._t;
+
+    var DuplicateVisibility = FormView.include({
+        /**
+         * Instantiate and render the sidebar if a sidebar is requested
+         * Sets this.sidebar
+         * @param {jQuery} [$node] a jQuery node where the sidebar should be inserted
+         * $node may be undefined, in which case the FormView inserts the sidebar in a
+         * div of its template
+         **/
+        render_sidebar: function($node) {
```

Comment:
you replace super here, this breaks modularity. Implement your behavior by removing the button afterwards, or by patching `is_action_enabled`. But you need to call super.